Sources/TestingMacros/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+cmake_minimum_required(VERSION 3.28)
+
+if(POLICY CMP0157)
+  cmake_policy(SET CMP0157 NEW)
+endif()
+
+project(TestingMacros
+  LANGUAGES Swift)
+
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
+
+add_compile_definitions(_CRT_SECURE_NO_WARNINGS)
```

Comment:
It is correct to define `_CRT_SECURE_NO_WARNINGS`, but also _who deprecates `fopen()`???_

Reply:
(But also also: Windows-only, right?)

Reply:
I should put it in an `if(WIN32)`.

Reply:
> but also who deprecates fopen()???

Lol.  Microsoft.  _I was there, Gandalf..._